packages/core/lib/commands/help.js

```diff
@@ -52,12 +55,19 @@ const command = {
       commandHelp = await commandHelp(options);
     }
 
-    console.log(`\n  Usage:        ${commandHelp.usage}`);
+    const allowedGlobalOptions = commandHelp.allowedGlobalOptions.map(tag => commonOptions[tag].option ? commonOptions[tag] : "");
```

Comment:
Same as above:

```suggestion
    const allowedGlobalOptions = commandHelp.allowedGlobalOptions
      .filter(tag => tag in commonOptions)
      .map(tag => commonOptions[tag].option);
```

Reply:
also you're calling it `commonOptions` here but `commandOptions` above - which is it?

Reply:
Renamed!